src/fields/hero.ts

```diff
@@ -138,12 +149,7 @@ export const hero: Field = {
       overrides: {
         admin: {
```

Comment:
@paulpopus I say we add `appearances: false` to this linkGroup since none of the heros will be using any of the other button appearances. 

This will require some updates on the front-end e.g. specifying that all links for the all the heros use the default button appearance.